lib/rucio/rse/protocols/gfal.py

```diff
@@ -28,7 +28,7 @@
 
 try:
     import gfal2  # pylint: disable=import-error
-except:
+except:  # noqa: E722
```

Comment:
```suggestion
except Exception:
```

This way you can avoid having the noqa, while also avoiding catching possible system exceptions that we might not want to catch: https://docs.astral.sh/ruff/rules/bare-except/